packages/lib-classifier/src/plugins/drawingTools/components/RotateRectangle/RotateRectangle.js

```diff
@@ -0,0 +1,160 @@
+  const y_top = y_center - height / 2
+  const y_bottom = y_center + height / 2
+  const xRotationHandle = x_center + width / 2 + BUFFER
+
+  function onHandleDrag(coords) {
+    mark.setCoordinates(coords)
+  }
+
+  function onRotateDrag(e) {
+    const angle = mark.getAngle(x_center, y_center, e.x, e.y)
+    mark.setCoordinates({ x_left, x_right, y_top, y_bottom, angle })
+  }
+
+  return (
+    <g onPointerUp={active ? onFinish : undefined}>
+      <rect x={x_left} y={y_top} width={width} height={height} />
+      <rect
+        x={x_left}
+        y={y_top}
+        width={width}
+        height={height}
+        strokeWidth={GRAB_STROKE_WIDTH / scale}
+        strokeOpacity='0'
+      />
+      {/* Rotate Handle */}
+      {active && (
+        <g>
+          <line
+            x1={x_center}
+            y1={y_center}
+            x2={xRotationHandle}
+            y2={y_center}
+            strokeWidth={guideWidth}
+            strokeDasharray={GUIDE_DASH}
+          />
+          <RotateHandle
+            dragMove={onRotateDrag}
+            scale={scale}
+            x={xRotationHandle}
+            y={y_center}
+          />
+        </g>
+      )}
+
+      {active && (
+        <DragHandle
+          scale={scale}
+          x={x_left}
+          y={y_top}
+          dragMove={(e, d) =>
+            onHandleDrag({
+              x_left: x_left + d.x,
+              x_right: x_right,
+              y_top: y_top + d.y,
+              y_bottom: y_bottom,
+              angle: angle
+            })
+          }
+        />
+      )}
+      {active && (
+        <DragHandle
+          scale={scale}
+          x={x_right}
+          y={y_bottom}
+          dragMove={(e, d) =>
+            onHandleDrag({
+              x_left: x_left,
+              x_right: x_right + d.x,
+              y_top: y_top,
+              y_bottom: y_bottom + d.y,
+              angle: angle
+            })
+          }
+        />
+      )}
+      {active && (
+        <DragHandle
+          scale={scale}
+          x={x_left}
+          y={y_bottom}
+          dragMove={(e, d) =>
+            onHandleDrag({
+              x_left: x_left + d.x,
+              x_right: x_right,
+              y_top: y_top,
+              y_bottom: y_bottom + d.y,
+              angle: angle
+            })
+          }
+        />
+      )}
+      {active && (
+        <DragHandle
+          scale={scale}
+          x={x_right}
+          y={y_top}
+          dragMove={(e, d) =>
+            onHandleDrag({
+              x_left: x_left,
+              x_right: x_right + d.x,
+              y_top: y_top + d.y,
+              y_bottom: y_bottom,
+              angle: angle
+            })
+          }
+        />
+      )}
+      {children}
+    </g>
+  )
+}
+
+RotateRectangle.propTypes = {
+  /**
+    Modal active state.
+  */
+  active: PropTypes.bool,
+  /**
+    node
+  */
+  children: PropTypes.node,
+  /**
+    object
+  */
+  mark: PropTypes.object.isRequired,
```

Comment:
Not a blocking change, but you can use PropTypes.shape() here to define individual properties for the mark.